sys/crypto/aes.c

```diff
@@ -32,10 +32,6 @@
  * @}
  */
 
-#include <stdio.h>
-#include <stdarg.h>
-#include <string.h>
```

Comment:
You still need `string.h` for `memset()`

Reply:
oops, yes. Sorry, I'm afk for two hours